aiida/cmdline/commands/cmd_calcjob.py

```diff
@@ -228,7 +228,11 @@ def calcjob_outputls(calcjob, path, color):
 @options.OLDER_THAN(default=None)
 @options.COMPUTERS(help='include only calcjobs that were ran on these computers')
 @options.FORCE()
-def calcjob_cleanworkdir(calcjobs, past_days, older_than, computers, force):
+@options.EXIT_STATUS()
+@click.option(
+    '--include-cleaned', is_flag=True, default=False, help='Included calcjobs that have been cleaned previously'
+)
```

Comment:
I am still not sure we should complicate it here for the cases we have not encountered yet. I would prefer to add this option only when the performance issue hit. I let @ramirezfranciscof involved for some advice. 

If we agree this option is necessary then I think it better be `@options.INCLUDE_CLEANED` to compatible the code style we have.

Reply:
Defining an option as a reusable option is only really necessary if it is used more than once. The idea of having them in `options` module is to make it easy to reuse so similar options look similar across the CLI. If there is an option that is a one-off, it is fine to keep it defined here in place.